corehq/form_processor/models.py

```diff
@@ -263,6 +263,13 @@ def is_deleted(self):
         return self.state & self.DELETED == self.DELETED
 
     @property
+    def doc_type(self):
+        from corehq.form_processor.backends.sql.dbaccessors import doc_type_to_state
+        if self.is_deleted:
+            return 'XFormIntance' + DELETED_SUFFIX
```

Comment:
this should probably be fixed before merge right? won't this mean we'll have incorrect doc types for some deleted forms in elastic?